src/org/jitsi/impl/neomedia/stats/MediaStreamStats2Impl.java

```diff
@@ -140,6 +151,24 @@ public void rtcpReceiverReportReceived(long ssrc, int fractionLost)
         {
             getSendStats(ssrc).rtcpReceiverReportReceived(fractionLost);
         }
+
+        this.cleanSendStatsOld();
+    }
+
+    /**
+     * Clean old send stats.
+     */
+    private void cleanSendStatsOld()
+    {
+        long now = System.currentTimeMillis();
+        this.sendSsrcStatsToClean.entrySet().stream().forEach(entry ->
```

Comment:
This probably has negligible overhead but since this may run often, shall we check if the map is empty?

Reply:
I'm adding it.
But this is not running so often, it's running on every rtcp report.